starport-metrics/app/main.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+from flask import Flask
+import json
+import os
+from metrics.metrics import render_metrics, label_metrics
+
+app = Flask(__name__)
+
+
+@app.route('/metrics')
+def metrics():
+    return render_metrics(build_target_metrics("/starport"))
+
+
+def build_target_metrics(starport):
+    metrics = {}
+    targets = os.listdir(starport)
+    for target in targets:
+        try:
+            with open(os.path.join(starport, target, "meta", "metadata.json")) as json_data:
+                d = json.load(json_data)
+                item_metrics = label_metrics(d, {"target_id": target})
+                metrics.update(item_metrics)
+        except:
+            item_metrics = {"missing_metadata": True}
```

Comment:
Maybe instead of a `missing_metadata` metric that is sometimes present, it would be better to have a `metadata_present` metrics that is always there, and is either True or False depending.